### PR TITLE
[CAI-21] Adding speakers and opening CFP

### PR DESCRIPTION
**Pull request by aameen79**
Adding Keynote speakers and opening CFP

### CODE REVIEW

File: content/events/2021-cairo/welcome.md [hunk 1]
```diff
@@ -6,19 +6,17 @@ Description = "DevOpsDays Cairo 2021"
...
.../LXvNRtLa+x2C8yoemUL+PjmEBDVy13EfqTLwZHZrBZMvmEFK6UztorZygOPA83Xf+RueieTo75wlT02SJJsqXVY9ytU1sL5F2Pidb/opW2qVqRfIIoYMm4abixnR9qhH1B3ArmuD+Z7T5x9UMfJRNUtYv+2xyJfAE8KQp01VTkL9gOPUJNGH/w4YteAh4gEHwqQf8Cs0b/Zep4h1oM5Pb0EBVzSdo8MkNqTPZ1OWiEjSKH6L0A1iSZVRpSpSK6aVjhMUPiEEorTGIIGQxUgRBspQ8BdKEKk1AIlnC1x4zvU0p0DlJd7H1XhVPo5VGJ4GPRh7bY8bqEVOGu0wN1io0NAzpLvt3M2X6E1tqP277Xsp4VdRx4K9olynQ4Naf0ODRZY2/9QFzF3xmhrQAvmHHTEZcFO+5IT0na0rn/0WdnVXX5etGzV64qDyFJCmddIpISsyEvCrJYrRRHrH/bZEAy1kg7QXyEqY7OgtLCh0FHSb/s67LJOe3uH8JXGG/LwIv2bJ9b2OfnfZTE4Fjpgj3NYzavKnS9drWnUDTn66y4/1gnf0WTLG+NrL978CvgV/a8+uepJcz7Fsd5Q27WVyw7fTOTfenz61LnnOeGFWxNDHpa50QKqZim9lqkemqTUZFklj+av1jFVY7uj1yupB1CaFHKpEsbjsjmrK1LqbhLK+Txp2IEu0J+pQp0+4aS/+2GbRH0Br3mikzjHs2uIY/NhX6DzvOqBuhY6/7W+CPjI+ULoHfAz9H06BO23NGibb9C1O1j6EBqQtWMp0b49ZFKDgC7EYu7OYG2wLZgmnYzpdMICWwRMXJLey9gJZHFpx5p/oTjf8voSWiB809cBOaklSiEe9X0FLOf6MBnANmROsGJ3sZ9CU9CbxvjyemYl8F3gTutWNfa48toFVIr6KR+N4677dnS/z99vzrzVVQj3xO0BLRA6ZE93OB19nDOcxxlp+xg4wcn1M2GYY2bOtvvWZ3BZbDbzbNzWyhQZWz+YuXWTt1KUWj3vUMiVWGK5NyNLiTNM78KoMKommGA1AlcKph8GcYdN2v7LGvK2QKhqufamPbZjs1L3Qcx3Ecx3Ecx3Ecx3Ecx3Ecx3Ecx3Ecx3Ecx3Ecx3Ecx3Ecx3Ecx5k4/gP4QWUQowWmEwAAAABJRU5ErkJggg==" style="height: 50px;">&nbsp;</p>
```
Comment (glasnt) [inline content/events/2021-cairo/welcome.md:12]: Is this change intentional? It's made the welcome page render incorrectly (especially around sponsor listings) https://deploy-preview-10330--devopsdays-web.netlify.app/events/2021-cairo/welcome/

**Comment by aameen79** [comment]
@glasnt Fixed
